students/JamesTraub/lesson03/mailroom.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# James Traub 04-17-18
+# Lesson 3 - Mailroom Lab Exercise, Part 1
+
+
+from datetime import date
+today = str(date.today())
+
+if __name__ == "__main__":
+    print('')
```

Comment:
this block should go at the bottom of your script and call your "main" from here (aka your entry point)